aesd-char-driver/main.c

```diff
@@ -21,8 +21,10 @@
 #include "linux/string.h"
 #include <linux/fs.h> // file_operations
 #include "aesdchar.h"
+#include "aesd_ioctl.h"
 int aesd_major =   0; // use dynamic major
 int aesd_minor =   0;
+int total_buffer_size = 0; // for llseek size 
```

Comment:
@sachininja what assumption does adding a global here make regarding the number of devices supported by the driver?